admin/schemas.php

```diff
@@ -153,6 +176,19 @@ protected function html_json(Schema $schema) {
         $form->addButton('import', $this->getLang('btn_import'));
         $form->addHTML('<p>' . $this->getLang('import_warning') . '</p>');
         $form->addFieldsetClose();
+
+        $form->addFieldsetOpen($this->getLang('csvexport'));
```

Comment:
Using a different lang string here might be more consistent with the other labels on that page. Maybe something like `Export data as CSV` ?